ddtrace/appsec/_iast/_taint_tracking/native.cpp

```diff
@@ -41,23 +41,25 @@ static PyMethodDef AspectsMethods[] = {
     { nullptr, nullptr, 0, nullptr }
 };
 
-static struct PyModuleDef aspects = { PyModuleDef_HEAD_INIT,
-                                      .m_name = PY_MODULE_NAME_ASPECTS,
-                                      .m_doc = "Taint tracking Aspects",
-                                      .m_size = -1,
-                                      .m_methods = AspectsMethods };
+// Mark the module as used to prevent it from being stripped.
+static struct PyModuleDef aspects __attribute__((used)) = { PyModuleDef_HEAD_INIT,
```

Comment:
Why is __attribute__((used)) forced on these methods and not on the rest? Does it cause any errors if it's not used?

Reply:
The smoke test complained that it can't find `ops` and `aspects` modules in this file.